src/assets/scripts/math/circle.js

```diff
@@ -67,5 +68,35 @@ export const radians_normalize = (radians) => {
     if (radians >= 0) {
       return radians % (tau());
     }
+
     return tau() + (radians % tau());
 };
+
+/**
+ * Calculate the distance between two lat/long coordinates in km
+ *
+ * see: http://stackoverflow.com/questions/27928/calculate-distance-between-two-latitude-longitude-points-haversine-formula
+ * for more information on the math here
+ *
+ * @function distanceToPoint
+ * @param startLatitude {number}
+ * @param startLongitude {number}
+ * @param endLatitude {number}
+ * @param endLongitude {number}
+ * return {number}
+ */
+export const distanceToPoint = (startLatitude, startLongitude, endLatitude, endLongitude) => {
+    const EARTH_RADIUS_KM = 6371;
+    const startLatitudeRadians = degreesToRadians(startLatitude);
+    const endLatitudeRadians = degreesToRadians(endLatitude);
+    const distanceLatitude = degreesToRadians(startLatitude - endLatitude);
+    const distanceLongitude = degreesToRadians(startLongitude - endLongitude);
+
+    // TODO: what is actually getting set to `a` here? and what does `a` represent?
+    const a = Math.pow(Math.sin(distanceLatitude / 2), 2) +
+        (Math.cos(startLatitudeRadians) * Math.cos(endLatitudeRadians) * Math.pow(Math.sin(distanceLongitude / 2), 2));
+    // TODO: what is actually getting set to `c` here? and what does `c` represent?
+    const c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1 - a));
+
+    return c * EARTH_RADIUS_KM;
```

Comment:
@erikquinn all the links I found on the Haversine formula don't actually spell out what `a` and `c` here mean (that I could find/understand, I don't speak math very well).  What do these letters mean?

Also, I found some interesting stuff on the law of cosines that could be used instead of the Haversine, is there a specific reason we _are_ using that here?  Any reason why we're not using the law of cosines?  Or is it just was here?